### PR TITLE
1.30 "Austria" update

### PR DESCRIPTION
**Pull request by HerrX2000**
Work in progress Austria update for the cw tools eu4 config.

### CODE REVIEW

File: effects.cwt [hunk 1]
```diff
@@ -2986,4 +2990,57 @@ alias[effect:native_policy] = enum[native_policies]
...
+alias[effect:override_country_name] = localisation
+
+## scope = country
+alias[effect:add_revolutionary_zeal] = int
+
+## scope = country
+alias[effect:add_years_of_estate_land_income] = float
+
+## scope = country
+alias[effect:kill_mercenary_leader] = this
```
Comment (tboby) [inline effects.cwt:3012]: The `THIS` is a bit weird, but yeah, that does seem to be how they use it

File: triggers.cwt [hunk 2]
```diff
@@ -4521,3 +4521,32 @@ alias[trigger:recent_treasure_ship_passage] = yes
 ## scope = province
 ###Checks whether the province is getting an expelled minority
 alias[trigger:province_getting_expelled_minority] = bool
+
+#1.30
+
+## scope = country 
+num_of_owned_areas = 1
```
Comment (tboby) [inline triggers.cwt:4528]: `int`? :)

File: triggers.cwt [hunk 3]
```diff
@@ -4521,3 +4521,32 @@ alias[trigger:recent_treasure_ship_passage] = yes
...
+
+## scope = province
+is_revolution_center = yes
```
Comment (tboby) [inline triggers.cwt:4531]: `bool`?

File: triggers.cwt (hunk 2 above)
Comment (HerrX2000) [reply triggers.cwt:4528]: ups

File: triggers.cwt (hunk 3 above)
Comment (HerrX2000) [reply triggers.cwt:4531]: ups

**Comment by HerrX2000** [comment]
Not 100% complete but all the stuff included works well and improves the experience with 1.30 a lot.

Besides the 1.30 additions, I've added set_value to government_attribute to support custom attributes.

**Comment by HerrX2000** [comment]
I feel comfortable now to propose the PR to be merged. I've find more stuff, I'll make another PR.

**Pull request merged**